### PR TITLE
Add support for failover, geolocation, and latency routing policy to terraform-aws-route53-record

### PR DESCRIPTION
**Pull request by JonRoma**
This very old pull request may prove useful if IAMU pursues multi-region support for the Shibboleth IdP. It's being pushed under a more meaningful branch name.

### CODE REVIEW

**Comment by JonRoma** [comment]
@ddriddle, please note that the default branch is now "main" if you have a working copy of this code checked out.

File: examples/failover/terragrunt.hcl [hunk 1]
```diff
@@ -0,0 +1,24 @@
+include {
+  path = "${find_in_parent_folders()}"
+}
+
+terraform {
+  source = "git@github.com:techservicesillinois/terraform-aws-route53-record?ref=v3.0.0"
```
Comment (cmaturi) [inline examples/failover/terragrunt.hcl]: Update tag version to latest - V3.0.1

File: examples/geolocation/terragrunt.hcl [hunk 2]
```diff
@@ -0,0 +1,24 @@
+include {
+  path = "${find_in_parent_folders()}"
+}
+
+terraform {
+  source = "git@github.com:techservicesillinois/terraform-aws-route53-record?ref=v1.0.0"
```
Comment (cmaturi) [inline examples/geolocation/terragrunt.hcl]: Update tag version to latest - V3.0.1

**Review by cmaturi — COMMENTED** [review]
Please update the terraform-aws-route53-record to latest tag version - V3.0.1

File: examples/geolocation/terragrunt.hcl (hunk 2 above)
Comment (JonRoma) [reply examples/geolocation/terragrunt.hcl]: Removed tag reference altogether.

**Comment by JonRoma** [comment]
@ddriddle, please look at this at your convenience. I made a few minor changes to the example an to the `README.md`.

Unless you prefer otherwise, I plan to squash all of the commits into a single commit.

**Comment by JonRoma** [comment]
@cmaturi, @ddriddle , @kwessel, @eccoleman, please review this at your convenience.

**Review by eccoleman — APPROVED** [review]

**Pull request merged**